addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/terminate-launch-dependency/terminate-launch-dependency.js

```diff
@@ -116,6 +117,28 @@ class TerminateLaunchDependency extends StepBase {
             msg: `Record deletion failed with error - ${error.message}`,
           });
         }
+        // Revoke EC2 security group rule with ALB security group ID
+        try {
+          const albSecurityGroup = JSON.parse(deploymentItem.value).albSecurityGroup;
+          const instanceSecurityGroup = _.get(environmentOutputs, 'InstanceSecurityGroupId', '');
+          const updateRule = {
+            fromPort: 443,
+            toPort: 443,
+            protocol: 'tcp',
+            groupId: albSecurityGroup,
+          };
+          await environmentScCidrService.revokeIngressRuleWithSecurityGroup(
+            requestContext,
+            envId,
+            updateRule,
+            instanceSecurityGroup,
+          );
+        } catch (error) {
+          // Don't fail the termination if revoke fails
+          this.print({
+            msg: `Security group rule revoke failed with error - ${error.message}`,
+          });
+        }
```

Comment:
Didn't see a corresponding unit test change for this.

Reply:
Incorporated the suggestion and updated the PR